stubs/pyinstaller/@tests/stubtest_allowlist.txt

```diff
@@ -9,9 +9,14 @@ PyInstaller.__main__.run_makespec
 PyInstaller.utils.hooks.conda.lib_dir
 
 # A mix of modules meant to be private, and shallow incomplete type references for other modules
-PyInstaller\.building\.\w+?
 PyInstaller.building.build_main.*
 PyInstaller.building.datastruct.unique_name
+PyInstaller.building.icon.*
+PyInstaller.building.makespec.*
+PyInstaller.building.osx.*
+PyInstaller.building.splash_templates.*
+PyInstaller.building.templates.*
+PyInstaller.building.utils.*
```

Comment:
```suggestion
PyInstaller.building.icon
PyInstaller.building.makespec
PyInstaller.building.osx
PyInstaller.building.splash_templates
PyInstaller.building.templates
PyInstaller.building.utils
```

Stubtest still passes with this change (I tested locally). If the module itself is missing, it only results in one stubtest error, so there's no need for regexes here. Avoiding regexes means that we'll get a helpful "unused allowlist" error if we ever add stubs for one of these modules, and we won't necessarily be accidentally allowlisting anything _in_ the module if we ever add stubs for that module.

We should probably get rid of the `.*`s in some of the other entries in this allowlist, too.

Reply:
I cleaned up the list further. The section titled "A mix of modules meant to be private, and shallow incomplete type references for other modules" is a bit suspect to me, too, but I don't want to delve into that now.